### PR TITLE
D8CORE-3686: Intranet styles.

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW
Resolved questions
- Favicon - added in the Theme settings as part of the site set up.
- Admin site - styles excluded.

# Summary
- Intranet styles
- Styles should not be included on the /admin/ pages.

# Needed By (Date)
- 4/7

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Verify the following:
   -  Brand bar is black
   - Body color is #f4f4f4
   - local footer is #eaeaea
   - super footer is #ffffff
   - Add content to the 3rd column in the Local footer. First `<p>` should say "Intranet" which will give you the bold and the icon. The second `<p>` would be the disclaimer.

# Affected Projects or Products
- Intranet project

# Associated Issues and/or People
- D8CORE-3686

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_intranet/stanford_intranet.module [hunk 1]
```diff
@@ -18,7 +18,7 @@ use Symfony\Component\Finder\Finder;
  * Implements hook_page_attachments().
  */
 function stanford_intranet_page_attachments(array &$attachments) {
-  if (\Drupal::state()->get('stanford_intranet', FALSE)) {
+  if ((\Drupal::state()->get('stanford_intranet', FALSE)) && (\Drupal::service('router.admin_context')->isAdminRoute() == FALSE)) {
```
Comment (pookmish) [inline modules/stanford_intranet/stanford_intranet.module:21]:
```suggestion
  if ((\Drupal::state()->get('stanford_intranet', FALSE)) && !\Drupal::service('router.admin_context')->isAdminRoute()) {
```

File: modules/stanford_intranet/lib/scss/base/intranet.scss [hunk 2]
```diff
@@ -1 +1,39 @@
 @charset "UTF-8";
+@import "./stanford_intranet.config";
+
+body {
```
Comment (pookmish) [inline modules/stanford_intranet/lib/scss/base/intranet.scss:4]: is `body` necessary here?

**Review by pookmish — CHANGES_REQUESTED** [review]
Just 1 change and 1 question

File: modules/stanford_intranet/lib/scss/base/intranet.scss (hunk 2 above)
Comment (jenbreese) [reply modules/stanford_intranet/lib/scss/base/intranet.scss:4]: I found a better option. :) Thank you for pointing it out.

**Review by pookmish — APPROVED** [review]
Thank you!

**Pull request merged**